interface/new/new_comprehensive.php

```diff
@@ -691,6 +692,10 @@ function end_group()
                           <label class='col-form-label col-md-1 mb-2 required'><?php echo xlt('Subscriber Address'); ?>:</label>
                           <div class="col-md-5 mb-2">
                             <input type='entry' class='form-control' size='25' name='i<?php echo attr($i); ?>subscriber_street' value="<?php echo attr($result3["subscriber_street"] ?? ''); ?>" onchange="capitalizeMe(this);" />
+                          </div>
+                          <label class='col-form-label col-md-1 mb-2 required'><?php echo xlt('Subscriber Apt/Suite/Other'); ?>:</label>
```

Comment:
it looks like in the practice settings there are 2 address lines. Should we just keep with that (ie. have 2 Address lines which would then be generic and user could use them for whatever they want) rather than a field specifically for apt/suite)? This would also make more sense for international use (ie. more generic/flexible, the better).

Reply:
btw, the 2 lines will also line up better with David's planned schema for the generic address table:
https://www.open-emr.org/wiki/images/4/4f/Projects_-_03_-_03_-_Demographic_Improvements_-_Table_Schema_2021-12-10.png